### ISSUE TITLE
Local install directory?

### ISSUE THREAD

**Issue by kyuucr** [ozonos/atom-launcher#19]
I have an issue about the install directory, from `make install` it will install to gnome shell root dir `/usr/share/gnome-shell/extensions` which will need root access and users can't remove it from Gnome Tweak Tool. I would like to change it to local user's extension dir `~/.local/share/gnome-shell/extensions/` but seeing as this extension and all of extensions on the extension set are intended for Ozon OS, it is making sense that it should be installed in root dir.

What do you guys think? I'm thinking of including both options to install in local and root dir. If we agreed on this, what is the default install action if we run `make install`? Local or root dir?

**Comment by me4oslav** [comment]
I agree that we can have a local/root flag for make install. We will have local flag if we choose root t be the stock and vice versa. However I dunno what should be the default. Our RPMS want root installation, but extensions.gnome.org want local one. What do we do?

**Comment by kyuucr** [comment]
Can the RPMS run `make install-root` for installing to root? I'm more
inclined to make the default `make install` to install to local.

Muhammad Iqbal CR

**Comment by me4oslav** [comment]
No clue. @satya164 can they?

**Comment by satya164** [comment]
Technically they can. But that would be non-standard. May be we can detect and do something in Makefile.

**Comment by kyuucr** [comment]
how about environment variable injection: `INSTALLROOT=true make install`?
